build/prbenchmarks.runtime.linux_arm64.config.yml

```diff
@@ -9,6 +9,15 @@ components:
         arguments:
             --application.options.outputFiles ./artifacts/tests/coreclr/Linux.arm64.Release/tests/Core_Root/
 
+    runtime-libs:
```

Comment:
Personally I would prefer something easier, like `libraries` but I don't deal with the runtime so if you prefer `runtime-libs` I'll merge.

Reply:
Renamed, thanks